errors.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package rate
+
+import "errors"
+
+var (
+	// ErrCapacityReached indicates that the store cannot store any additional quotas.
+	ErrCapacityReached = errors.New("capacity reached")
```

Comment:
```suggestion
	// ErrLimiterFull is returned by Limiter.Allow when the limiter cannot store any additional quotas.
	ErrLimiterFull     = errors.New("limiter full")
```
Just a suggestion. I tend to say something is full rather than something has reached its capacity. (_I'm full_ vs _Capacity reached in my stomach_)
 
Also, the current description mentions a "store" but there is no exported store. 
> ErrCapacityReached indicates that the **store** cannot store any additional quotas.